### PR TITLE
SE-141: Polish for list page

### PR DESCRIPTION
**Pull request by bkemper**
Refer to [SE-141](https://jira.int.messagesystems.com/browse/SE-141)

### What Changed

- [x] Fix content width for the border
- [x] Change title to 'Recently Triggered'
- [x] Tooltip with 'View Details' eyeball icon
- [x] Vertical alignment on eyeball
- [x] Use default padding around rows and header
- [x] Filter box suggestions
- [x] Semi bold weight on alert name links
- [x] Tooltip on delete icon
- [x] Change Status label to Active

### How To Test
 - Confirm filter suggestions are displaying and working correctly
 - Hover over the view and delete icon to see the tooltip

### Todo
 - [x] Tooltip is too wide
 - [ ] ~Search on metric friendly name~ Moved to after work ticket
 - [x] Check templates bolding
 - [x] Recently Triggered -> Recent Alerts

### CODE REVIEW

File: src/pages/alerts/ListPageNew.js [hunk 1]
```diff
@@ -73,7 +73,9 @@ export class ListPageNew extends Component {
                   <h3>{alert.name}</h3>
                 </Panel.Section>
                 <Panel.Section className = {styles.Footer}>
-                  <Button flat component={Link} to = {`/alerts-new/details/${alert.id}`}><RemoveRedEye className = {styles.Icon}/></Button>
+                  <Tooltip content='View Details' width='100px'>
```
Comment (bkemper) [inline src/pages/alerts/ListPageNew.js]: Are we using dark Tooltips?

File: src/pages/alerts/ListPageNew.js [hunk 2]
```diff
@@ -73,7 +73,9 @@ export class ListPageNew extends Component {
...
+                    <Button flat component={Link} to = {`/alerts-new/details/${alert.id}`}><RemoveRedEye className = {styles.Icon}/></Button>
```
Comment (bkemper) [inline src/pages/alerts/ListPageNew.js]: Prettier?

File: src/pages/alerts/components/AlertCollection.module.scss [hunk 3]
```diff
@@ -4,19 +4,12 @@
...
-
-.Title {
-  padding: spacing(larger) 2rem;
+.AlertNameLink {
+  font-weight: 600;
 }
 
 .FilterBox {
-  padding: spacing(larger) 2rem spacing(base);
+  padding: 1rem 1rem ;
```
Comment (bkemper) [inline src/pages/alerts/components/AlertCollection.module.scss]: Should these be `1rem` or `rem(14)`

File: src/pages/alerts/components/tests/AlertCollectionNew.test.js [hunk 4]
```diff
@@ -55,7 +55,7 @@ describe('TestCollection Component', () => {
 
   it('should handleDelete', () => {
     const actionCol = shallow(wrapper.instance().getRowData(props.alerts[0]).pop());
-    actionCol.find('button').simulate('click');
+    actionCol.dive().find('Button').simulate('click');
```
Comment (bkemper) [inline src/pages/alerts/components/tests/AlertCollectionNew.test.js]: Is `dive` necessary?

File: src/pages/alerts/ListPageNew.js (hunk 1 above)
Comment (kchu1337) [reply src/pages/alerts/ListPageNew.js]: We use both kinds  so I though we decided to leave them as light?

File: src/pages/alerts/components/AlertCollection.module.scss (hunk 3 above)
Comment (kchu1337) [reply src/pages/alerts/components/AlertCollection.module.scss]: What about `spacing(base)`?

File: src/pages/alerts/components/tests/AlertCollectionNew.test.js (hunk 4 above)
Comment (kchu1337) [reply src/pages/alerts/components/tests/AlertCollectionNew.test.js]: Yes, because it's a shallow render so only the tooltip itself is rendered.

**Comment by bkemper** [comment]
@kchu1337 LGTM.  I can't approve since I created the PR.

**Pull request closed without merging**